### PR TITLE
Return focus to invoking context

### PR DESCRIPTION
**Pull request by hollsk**
adds a bit about when and where to return focus. 

also fixes an egregious spelling erreur

### CODE REVIEW

**Comment by jpw** [comment]
I thought it was "pronunciation"

**Comment by hollsk** [comment]
🤦‍♀ suggest it please, and I'll accept 💖

File: accessibility/common-remediations.md [hunk 1]
```diff
@@ -433,7 +434,7 @@ The language specified in the HTML `lang` attribute doesn't match the language u
 
 #### Why's it a problem?
 
-Words are pronounced differently in different languages. Screen reader software is designed to respect these pronouncation differences when it encounters web pages that specify their main language. 
+Words are pronounced differently in different languages. Screen reader software is designed to respect these pronounciation differences when it encounters web pages that specify their main language.
```
Comment (jpw) [inline accessibility/common-remediations.md]:
```suggestion
Words are pronounced differently in different languages. Screen reader software is designed to respect these pronunciation differences when it encounters web pages that specify their main language. 
```

File: accessibility/common-remediations.md [hunk 2]
```diff
@@ -472,7 +473,7 @@ An element on the page contains text in a different language to the rest of the
 
 #### Why's it a problem?
 
-Words are pronounced differently in different languages. Screen reader software is designed to respect these pronouncation differences when it encounters components on web pages that specify a change in language. 
+Words are pronounced differently in different languages. Screen reader software is designed to respect these pronounciation differences when it encounters components on web pages that specify a change in language.
```
Comment (jpw) [inline accessibility/common-remediations.md]:
```suggestion
Words are pronounced differently in different languages. Screen reader software is designed to respect these pronunciation differences when it encounters components on web pages that specify a change in language. 
```

**Comment by hollsk** [comment]
nvm i fix 😄

**Comment by hollsk** [comment]
🚀

**Pull request merged**